modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/document/library/DocumentLibraryDDMFormFieldTemplateContextContributorTest.java

```diff
@@ -209,10 +240,33 @@ public void testGetParametersForSignedInUser() {
 			documentLibraryDDMFormFieldTemplateContextContributor = _createSpy(
 				themeDisplay);
 
+		DDMFormFieldRenderingContext ddmFormFieldRenderingContext =
```

Comment:
@JanisZhang Why do you need to add this logic when there's already `_setUpItemSelector()` in `setUp()`?

Reply:
@dantewang sorry, this logic is incorrect, I don't want to add this commit in the pull... forgot to drop it.